### PR TITLE
Restore host IPv4 addresses used in original exercises

### PR DESCRIPTION
**Pull request by jafingerhut**
The original exercises all used 10.0.1.1 10.0.2.2 10.0.3.3, and some
of the P4 tutorial slides also use these, not 10.0.0.1 10.0.0.2
10.0.0.3 that the original version of basic.p4app and
basic_tunnel.p4app have been using.  It seems best to make the
p4app-exercises versions match the originals in this detail, since it
is visible in other tutorial materials.

### CODE REVIEW

**Comment by jafingerhut** [comment]
@theojepsen FYI - this is the updated version -- As Antonin pointed out, my previous PR was based on a slightly older than latest version of the p4app branch files.

File: p4app-exercises/basic.p4app/main.py [hunk 1]
```diff
@@ -50,19 +56,19 @@ def __init__(self, n, **opts):
...
     # Otherwise send the packet to another switch
     for j in range(1, N+1):
         if i == j: continue
         sw.insertTableEntry(table_name='MyIngress.ipv4_lpm',
-                            match_fields={'hdr.ipv4.dstAddr': ["10.0.0.%d" % j, 32]},
+                            match_fields={'hdr.ipv4.dstAddr': [hostIP(j), 32]},
                             action_name='MyIngress.ipv4_forward',
-                            action_params={'dstAddr': '00:00:00:00:ff:%02x' % j,
-                                              'port': getForwardingPort(i, j)})
+                            action_params={'dstAddr': hostMAC(j),
```
Comment (antoninbas) [inline p4app-exercises/basic.p4app/main.py]: should be `switchMAC` here (and with `switchMAC(j) != hostMac(j)`)

File: p4app-exercises/basic_tunnel.p4app/main.py [hunk 2]
```diff
@@ -63,10 +69,10 @@ def __init__(self, n, **opts):
     for j in range(1, N+1):
         if i == j: continue
         sw.insertTableEntry(table_name='MyIngress.ipv4_lpm',
-                            match_fields={'hdr.ipv4.dstAddr': ["10.0.0.%d" % j, 32]},
+                            match_fields={'hdr.ipv4.dstAddr': [hostIP(j), 32]},
                             action_name='MyIngress.ipv4_forward',
-                            action_params={'dstAddr': '00:00:00:00:ff:%02x' % j,
-                                              'port': getForwardingPort(i, j)})
+                            action_params={'dstAddr': hostMAC(j),
```
Comment (antoninbas) [inline p4app-exercises/basic_tunnel.p4app/main.py]: same

File: p4app-exercises/basic.p4app/main.py (hunk 1 above)
Comment (jafingerhut) [reply p4app-exercises/basic.p4app/main.py]: Good catch.  I may have it correct now.

File: p4app-exercises/basic_tunnel.p4app/main.py (hunk 2 above)
Comment (jafingerhut) [reply p4app-exercises/basic_tunnel.p4app/main.py]: fixed

**Pull request merged**